rds-aurora-mysql/spicepod.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1beta1
+kind: Spicepod
+name: supabase_quickstart
+datasets:
+  - from: mysql:[remote_table_path]
+    name: [local_table_name]
+    params:
+      mysql_host: [db-host]
+      mysql_port: "[db-port]"
+      mysql_db: [db-name]
+      mysql_user: [username]
+      mysql_pass_key: [password]
+      mysql_sslmode: preferred
```

Comment:
Add comment as to the default. If this is the default, remove.

Reply:
added